tgstation.dme

```diff
@@ -3322,34 +3338,53 @@
 #include "modular_skyrat\code\datums\status_effects\buffs.dm"
 #include "modular_skyrat\code\datums\status_effects\debuffs.dm"
+#include "modular_skyrat\code\datums\traits\negative.dm"
 #include "modular_skyrat\code\datums\traits\neutral.dm"
 #include "modular_skyrat\code\datums\weather\weather.dm"
 #include "modular_skyrat\code\datums\weather\weather_types\acid_rain.dm"
 #include "modular_skyrat\code\datums\weather\weather_types\ash_storm.dm"
+#include "modular_skyrat\code\datums\weather\weather_types\radiation_storm.dm"
 #include "modular_skyrat\code\datums\weather\weather_types\snow_storm.dm"
 #include "modular_skyrat\code\game\world.dm"
 #include "modular_skyrat\code\game\area\areas.dm"
 #include "modular_skyrat\code\game\area\Space_Station_13_areas.dm"
 #include "modular_skyrat\code\game\area\areas\mining.dm"
 #include "modular_skyrat\code\game\area\areas\ruins\lavaland.dm"
 #include "modular_skyrat\code\game\gamemodes\objective.dm"
+#include "modular_skyrat\code\game\gamemodes\brother\traitor_bro.dm"
+#include "modular_skyrat\code\game\gamemodes\changeling\changeling.dm"
+#include "modular_skyrat\code\game\gamemodes\changeling\traitor_chan.dm"
+#include "modular_skyrat\code\game\gamemodes\cult\cult.dm"
+#include "modular_skyrat\code\game\gamemodes\devil\devil_game_mode.dm"
+#include "modular_skyrat\code\game\gamemodes\dynamic\dynamic_rulesets_midround.dm"
+#include "modular_skyrat\code\game\gamemodes\dynamic\dynamic_rulesets_roundstart.dm"
+#include "modular_skyrat\code\game\gamemodes\monkey\monkey.dm"
+#include "modular_skyrat\code\game\gamemodes\revolution\revolution.dm"
+#include "modular_skyrat\code\game\gamemodes\traitor\traitor.dm"
+#include "modular_skyrat\code\game\machinery\prisonlabor.dm"
 #include "modular_skyrat\code\game\machinery\computer\arcade.dm"
+#include "modular_skyrat\code\game\machinery\computer\card.dm"
+#include "modular_skyrat\code\game\machinery\computer\crew.dm"
 #include "modular_skyrat\code\game\machinery\computer\arcade\minesweeper.dm"
 #include "modular_skyrat\code\game\machinery\pipe\construction.dm"
 #include "modular_skyrat\code\game\mecha\mecha_construction_paths.dm"
 #include "modular_skyrat\code\game\mecha\mecha_wreckage.dm"
 #include "modular_skyrat\code\game\mecha\combat\gygax.dm"
 #include "modular_skyrat\code\game\objects\items.dm"
+#include "modular_skyrat\code\game\objects\effects\landmarks.dm"
 #include "modular_skyrat\code\game\objects\items\BSRPD.dm"
 #include "modular_skyrat\code\game\objects\items\cards_ids.dm"
 #include "modular_skyrat\code\game\objects\items\kitchen.dm"
+#include "modular_skyrat\code\game\objects\items\religion.dm"
 #include "modular_skyrat\code\game\objects\items\weaponry.dm"
 #include "modular_skyrat\code\game\objects\items\circuitboards\machine_circuitboards.dm"
 #include "modular_skyrat\code\game\objects\items\devices\radio\electropack.dm"
-#include "modular_skyrat\code\game\objects\items\implants\implant_stealth.dm"
```

Comment:
Is this removed and put somewhere else or?

Reply:
Github being weird still I guess

Reply:
fixed fuck